analysed-packages/kmod/version-v29/kmod-v29-SPDX2TV.spdx

```diff
@@ -0,0 +1,7184 @@
+FileChecksum: MD5: 2076b233b634bb6220e293b50ae6f196
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: <text> Copyright (C) 2016 Intel Corporation. All rights reserved. </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/libkmod/libkmod-list.c
+SPDXID: SPDXRef-item522304
+FileChecksum: SHA1: c538b7bbb8a32fd47c6317d93828a0fd2df98d2e
+FileChecksum: SHA256: 4c5f39c0a4893027f7f6d47308e084e7986b438c3104dae610dbe206827c0816
+FileChecksum: MD5: 6a38d1bd835a841e75209ebfaa25820e
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: <text> Copyright (C) 2011-2013 ProFUSION embedded systems </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/libkmod/libkmod-index.h
+SPDXID: SPDXRef-item522284
+FileChecksum: SHA1: 0ea17631facb6b3909b8136c1a981b159363a56f
+FileChecksum: SHA256: de8d07f823627c62e235570ec3566ce46f4657f46f6d14147b35b55b2c6eaffd
+FileChecksum: MD5: 8c7ed4aa827d519e95cc49b40cd39840
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: <text> Copyright (C) 2011-2013 ProFUSION embedded systems </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/shell-completion/bash/kmod
+SPDXID: SPDXRef-item522332
+FileChecksum: SHA1: 3afa5fa14bed3ddf59769251fd885a8c7c3f5daa
+FileChecksum: SHA256: b6fcd713807c9a4eacc3100b58b5e89e7d6cbe1253da41f0c8dd1b1f0efbab7e
+FileChecksum: MD5: 3592da55a19f765ee35dffc6f8fa0215
+LicenseConcluded: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-lgpl-2.1-plus
+FileCopyrightText: <text> Copyright 2010 Ran Benita
+Copyright (C) 2013 Intel Corporation. All rights reserved. </text>
+
+##File
+
+FileName: kmod-29.tar.gz/kmod-29.tar/kmod-29/testsuite/COPYING
+SPDXID: SPDXRef-item522881
+FileChecksum: SHA1: 545f380fb332eb41236596500913ff8d582e3ead
+FileChecksum: SHA256: 6095e9ffa777dd22839f7801aa845b31c9ed07f3d6bf8a26dc5d2dec8ccc0ef3
+FileChecksum: MD5: a6f89e2100d9b6cdffcea4f398e37343
+LicenseConcluded: LicenseRef-License-of-GNU-Licenses
+LicenseComments: <text> The file contains the text of the LGPL-2.1-or-later. The text of the license is licensed under:
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+Thus we concluded License-of-GNU-Licenses as license of this file. </text>
+LicenseInfoInFile: LicenseRef-LGPL-2.1-or-later
+LicenseInfoInFile: LicenseRef-LGPL-2.1-only
+LicenseInfoInFile: LicenseRef-License-of-GNU-Licenses
+LicenseInfoInFile: LicenseRef-LGPL-2.1
+FileCopyrightText: <text> Copyright (C) 1991, 1999 Free Software Foundation, Inc.
+written by James Random Hacker. </text>
```

Comment:
Please remove this, this is not a real copyright statement. For further details please see my comment in the disclosure document